src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksDtoToPaymentWorksVendorConversionServiceImpl.java

```diff
@@ -131,6 +133,17 @@ private void populateNewVendorBankAccountAttributes(PaymentWorksVendor stgNewVen
             populateNewVendorBankAddressAttributes(stgNewVendor, pmwBankAccountDTO);
         }
     }
+    
+    protected String sanitizeBankAcctBankAccountNumberValue(String pmwBankAccountNumber) {
+        if (StringUtils.isNotBlank(pmwBankAccountNumber) && StringUtils.contains(pmwBankAccountNumber, DASH)) {
+            String sanitizedPmwBankAccountNumberValue = StringUtils.replace(pmwBankAccountNumber, DASH, StringUtils.EMPTY);
+            LOG.info("sanitizeBankAcctBankAccountNumberValue: the PaymentWorks bank account number value: " + pmwBankAccountNumber + " has been sanitized to: "
```

Comment:
The bank account number is sensitive.  I don't think we should log that data.  However, logging that we had to trim out hyphens would be good.

Reply:
Good point, thank you for catching that. Updating.